File: estructura_datos/arrays_dinamicos/ejemplo.py
def run():
    length = int(input("What is the length of the list? "))
    my_list = [None] * length

    for index, _ in enumerate(my_list):
        my_list[index] = int(
            input(f"Enter the number for the position {index}: "))

    print(my_list)


if __name__ == "__main__":
    run()

File: estructura_datos/arrays_dinamicos/ejemplo.js
// Do an interactive console with JS

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("What is the length of the array? ", (size) => {
  arr = Array(Number(size));
  for (let i = 0; i < Number(size); i++) {
    arr[i] = i;
  }
  console.log(arr);
  rl.close();
});

rl.on("close", () => process.exit(0));
